src/pages/trails/ballots/view/BallotView.vue

```diff
@@ -384,25 +384,24 @@ export default {
                                                     div(v-if="getPartOfTotal(option)") {{ getPartOfTotalPercent(option) }}%&nbsp
                                     div.linear-progress(v-if="displayWinner(ballot)")
                                         q-linear-progress(rounded size="6px" :value="getPartOfTotal(option)" color="$primary")
-                            q-item(v-if="ballot.status !== 'cancelled' && isBallotOpened(ballot)").capitalize.options-btn
-                                q-item-section.btn-wrapper
-                                    btn(
+                            q-item(v-if="ballot.status !== 'cancelled' && isBallotOpened(ballot)").options-btn
+                                q-btn(
+                                    no-caps
+                                    :color="userCanVote ? 'primary' : 'info'"
+                                    class="col"
                                     v-if="isAuthenticated"
                                     :disable="!userCanVote"
-                                    :labelText="$t(voteButtonText)"
-                                    btnWidth='220'
-                                    fontSize='16'
-                                    hoverBlue=true
-                                    @clickBtn="vote()"
-                                    )
-                                    btn(
+                                    :label="$t(voteButtonText)"
+                                    @click="vote()"
+                                )
+                                q-btn(
+                                    no-caps
+                                    color="red"
```

Comment:
The color red usually indicates an action that can't be undone or is otherwise critical, such as deleting something and big self-destruct buttons.
Because the cancel button closes the current ballot, we should change it to the `secondary` button pattern: `outline` button.